recipe/meta.yaml

```
@@ -99,10 +101,6 @@ outputs:
     requirements:
       run:
         - isce3 ={{ version }}=*_cuda
-    test:
-      imports:
-        - isce3
-        - isce3.cuda
 
```

Think we still want to test these somehow

Particularly in the CPU case where we can `import`. In the GPU case we can at least check if the module exists (without `import`ing)

```suggestion
    test:
      {% set modules = [
          "isce3",
          "isce3.cuda",
      ] %}
      imports:                  # [cuda_compiler_version != "None"]
        {% for m in modules %}  # [cuda_compiler_version != "None"]
        - {{ m }}               # [cuda_compiler_version != "None"]
        {% endfor %}            # [cuda_compiler_version != "None"]
      commands:
        {% for m in modules %}
        - python -c 'import pkgutil; import sys; sys.exit(pkgutil.find_loader("{module}") is None)'
        {% endfor %}

```